src/webextension/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+
+  "description": "WebCompat Go Faster Web Extension",
+  "manifest_version": 2,
+  "name": "webcompat-extension",
+  "version": "1.0",
+
+  "permissions": [
+    "tabs",
+    // TODO: Fill required URL for override rules the same as ua_overrides.js
+    // For tablog.js to have permission to inject script to web console.
+    // Example: "*://www.google.com.tw/*"
```

Comment:
> so far I don't see performance issue here.

Okay, cool, thanks. :)

> But we can set from ua_overrides.js before we install to gecko-dev by jake.

That sounds good. Do you want to implement that as well? Feels much nicer than having to maintain two lists of domain.